Testscripts/Linux/DM_HotAdd_Verify_udev.sh

```diff
@@ -13,6 +13,16 @@
 # Source constants file and initialize most common variables
 UtilsInit
 items=(SUBSYSTEM==\"memory\", ACTION==\"add\", ATTR{state}=\"online\")
+
+GetDistro
```

Comment:
I was told 'GetDistro' called inside UtilInit. Can you check if you still need another call here? @lixuemin2016

Reply:
@juhlee-microsoft 
Thank you for your comment, no need another call here, updated based on your comment.